example/server/integrations.py

```diff
@@ -783,19 +784,13 @@ def toml_integration():
 
 
 def scripts_integration(page_content: Optional[Dict]):
-    tags = [
-        # Google Analytics
-        """
-        <!-- Global site tag (gtag.js) - Google Analytics -->
-        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-53373928-6"></script>
-        <script>
-          window.dataLayer = window.dataLayer || [];
-          function gtag(){dataLayer.push(arguments);}
-          gtag('js', new Date());
-          gtag('config', 'UA-53373928-6');
-        </script>
-        """
-    ]
+    tag_paths = []
```

Comment:
Can we change `tag_paths` to `scripts` or something more representative? And initialize the list with the contents:
```
scripts = [
    # Google Analytics
    # <!-- Global site tag (gtag.js) - Google Analytics -->
    tag_paths.append(TemplateScript(url="https://www.googletagmanager.com/gtag/js?id=UA-53373928-6", is_async=True))
    tag_paths.append(TemplateScript(file_path="sep24_scripts/google_analytics.js"))    
]
```